Orange/data/domain.py

```diff
@@ -58,69 +69,101 @@ def __init__(self, source, destination):
 
 
 class Domain:
-    def __init__(self, attributes, class_vars=None, metas=None, source=None):
+    def __init__(self, attributes=(), class_vars=(), metas=(), source=None):
```

Comment:
For strict domain subsets one can already do, IMHO preferred:
```py
domain[list_of_vars]
```
where
```py
features = ['Feature 1', 'Target']
new = domain[features]
```
retains correct `attributes`, `class_vars`, and `metas`, i.e.:
```py
assert len(new.attributes) == len(new.class_vars) == 1
```